From 439737f5f507b3cde5f7618e54869735598b20fb Mon Sep 17 00:00:00 2001
From: Bill Nottingham <notting@redhat.com>
Date: Fri, 11 Nov 2011 12:16:41 -0500
Subject: [PATCH 101/109] Don't bother with stdin/stdout/stderr for
 rmmod/modprobe of scsi_wait_scan. (#735867) (cherry
 picked from commit
 c0e2c32eebea3c99807ca5f0505dcb020250a2a6)

---
 systemd/system/fedora-wait-storage.service |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/systemd/system/fedora-wait-storage.service b/systemd/system/fedora-wait-storage.service
index 48e92a5..fb53496 100644
--- a/systemd/system/fedora-wait-storage.service
+++ b/systemd/system/fedora-wait-storage.service
@@ -13,3 +13,6 @@ ExecStart=-/sbin/rmmod scsi_wait_scan
 Type=oneshot
 TimeoutSec=0
 RemainAfterExit=yes
+StandardInput=null
+StandardOutput=null
+StandardError=null
-- 
1.7.9.2

